app/.gitignore

```diff
@@ -0,0 +1,3 @@
+dist
+node_modules
+.env
```

Comment:
Why not add generated artifacts (such as the `.zip` package) to gitignore

Reply:
yes. you are right